addons/rose/app/styles/rose/_typography.scss

```diff
@@ -12,60 +12,26 @@ html {
 }
 
 body {
+  // Used by Rose::Tabs and '/' in breadcrumbs
+  // and link list panel
   font-family: typography.$family-primary;
 
-  // This is the only time we'll explicitly use pixels.  All other sizes are
-  // expressed in rem units.
-  font-size: typography.$base-font-size * 1px;
+  // these 3 are used across the app
   line-height: typography.line-height(m);
   color: var(--stark);
   background-color: var(--subtle);
 }
 
-input,
-textarea,
-button {
-  font-family: typography.$family-primary;
-}
-
-pre,
-code {
-  font-family: typography.$family-monospace;
-}
-
-h1,
-.h1 {
-  @include type-mixins.type(h1, bold);
-
-  margin-top: sizing.rems(m);
-  margin-bottom: sizing.rems(xs);
-}
-
-h2,
-.h2 {
-  @include type-mixins.type(h2, bold);
-
+.header {
   margin-top: sizing.rems(m);
   margin-bottom: sizing.rems(xs);
 }
 
-h3,
-.h3 {
-  @include type-mixins.type(h3, bold);
-
-  margin-top: sizing.rems(m);
-  margin-bottom: sizing.rems(xs);
-}
-
-p,
 .p {
   margin-bottom: sizing.rems(m);
 }
 
+// once we move to Hds::Link::Standalone we can remove this
 a {
   color: var(--action);
 }
-
-time {
```

Comment:
I opted to remove this style completely since HDS does not use it. While we haven't migrated `time` to Hds::Time yet, this would get removed anyway.